source/_patterns/00-atoms/components/orcid.mustache

```diff
@@ -1,4 +1,4 @@
-<span class="orcid">
+<span class="orcid" aria-label="An ORCID iD is a persistent digital identifier for researchers">
```

Comment:
Doesn't read like a label, but a description of what ORCID is...

Reply:
Doesn't work in this context. Would be appropriate to expand the spoken description for the login button though, which means in `journal` adding to:
- https://github.com/elifesciences/journal/blob/105132a10587eaf3b9f0b7d796a9766c0ba7d52c/test/Controller/AuthenticationTest.php#L29
- https://github.com/elifesciences/journal/blob/105132a10587eaf3b9f0b7d796a9766c0ba7d52c/test/Controller/AuthenticationTest.php#L149
and
- https://github.com/elifesciences/journal/blob/fd91c74ba4e4f060c3bfda759dc5402f6f5f1749/src/ViewModel/Factory/SiteHeaderFactory.php#L159

Ensure to update the json files for the [pattern library example pages](https://github.com/elifesciences/pattern-library/tree/develop/source/_patterns/04-pages) too, to reflect the updated text.

Still need to work out the working to announce the ORCID in the articles.